Compilation error during import of SBML semantic case 01355

When importing SBML semantic case 01355 with enabled sensitivity code, compilation fails with:

```
/home/dweindl/src/AMICI-devel/tests/SBMLTestModels/01355/deltasx.cpp:23:61: error: ‘dA__xdt’ was not declared in this scope; did you mean ‘A__x’?
   23 |                     deltasx[0] = -stau0*xdot_old0 + stau0*(-dA__xdt + xdot_old0) - sx0;
      |                                                             ^~~~~~~
      |                                                             A__x
```

`dA__xdt` should be defined in `xdot.h` which is not generated. Maybe because xdot = [0] ?

--- a/python/sdist/amici/de_export.py
+++ b/python/sdist/amici/de_export.py
@@ -1999,9 +1999,10 @@ class DEModel:
 
                 # need to check if equations are zero since we are using
                 # symbols
-                if not smart_is_zero_matrix(self.eq("stau")[ie]):
+                if not smart_is_zero_matrix(self.eq("stau")[ie]) \
+                        and not smart_is_zero_matrix(self.eq("xdot")):
                     tmp_eq += smart_multiply(
-                        (self.sym("xdot_old") - self.sym("xdot")),
+                        self.sym("xdot_old") - self.sym("xdot"),
                         self.sym("stau").T,
                     )
 

